signing_service/signing_service/signing_service.py

```diff
@@ -135,16 +135,16 @@ def _validate_arguments(self):
 
 
 def _parse_arguments() -> argparse.Namespace:
-
-    def get_secret_variable_wrapper(env_variable_name=None, use_file=False):
+    def get_secret_variable_wrapper(env_variable_name=None, use_file=False, base64_convert=False):
         def wrapper(**kwargs):
-            return GetSecretVariable(env_variable_name, use_file, **kwargs)
+            return GetSecretVariable(env_variable_name, use_file, base64_convert, **kwargs)
         return wrapper
 
     class GetSecretVariable(argparse.Action):
-        def __init__(self, env_variable_name, use_file, option_strings, dest, nargs=None):
+        def __init__(self, env_variable_name, use_file, base64_convert, option_strings, dest, nargs=None):
             self.env_variable_name = env_variable_name
             self.use_file = use_file
+            self.base64_convert = base64_convert
```

Comment:
Adding support for `base64_convert` looks like something that should be added in a separate commit. It's needed for the Ethereum key but it's an independent change.

Reply:
OK, I will add this as separate commit.